integration-tests/spring-pojo-tests/src/test/java/org/apache/servicecomb/demo/pojo/test/ConnectionEventWatcher.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.servicecomb.demo.pojo.test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.servicecomb.foundation.vertx.ClientClosedEvent;
+import org.apache.servicecomb.foundation.vertx.ClientConnectedEvent;
+
+import com.google.common.eventbus.Subscribe;
+
+public class ConnectionEventWatcher {
+  private final List<Integer> counters = new ArrayList<>();
+
+  public List<Integer> getCounters() {
+    return counters;
+  }
+
+  @Subscribe
```

Comment:
all eventloop thread will sync invoke this
it's not a good sample.

Reply:
what's this watcher purpose?

Reply:
only for test
```java
Assert.assertArrayEquals("check connection count change", new Integer[] {1, 0}, watcher.getCounters().toArray());
```
watcher will receive a connection connected event when consumer calling
event.getTotalConnectedCount() = 1
and when 
```java
SCBEngine.getInstance().destroy();
```
watcher will receive a connection close event
event.getTotalConnectedCount() = 0